src/collections/_documentation/learn/breadcrumbs.md

```diff
@@ -48,15 +48,15 @@ Type
 
 ## Recording Crumbs
 
-Manual breadcrumb recording is also available and easy to use.  This way breadcrumbs
-can be added whenever something interesting happens.  For instance it might make sense
+Manual breadcrumb recording is also available and easy to use.  In this way, breadcrumbs
```

Comment:
This was already correct

Reply:
will push a new commit to correct it again 👍